pkgs/stdenv/generic/default.nix

```diff
@@ -202,6 +205,32 @@ let
 
           '' + ((builtins.getAttr reason remediation) attrs));
 
+      metaTypes = with lib.types; {
+        # These keys are documented
+        description = str;
+        longDescription = str;
+        branch = str;
+        homepage = str;
+        downloadPage = str;
+        license = either (listOf lib.types.attrs) lib.types.attrs;
+        maintainers = listOf str;
+        priority = int;
+        platforms = listOf str;
+        hydraPlatforms = listOf str;
+        broken = bool;
+
+        # Weirder stuff that doesn't appear in the documentation?
+        version = str;
+        updateWalker = bool;
```

Comment:
`updateWalker` seems to be [documented][1].

[1]: <https://github.com/NixOS/nixpkgs/blob/7f7ebe5bf6ecb6fcb3c5d603c8f2e5b927ad0750/doc/meta.xml#L238-L246>